detection-bot/README.md

```diff
@@ -0,0 +1,25 @@
+# Nethermind Bot Deployments Agent
```

Comment:
```suggestion
# Nethermind Bot Deployments Bot
```
Forta has migrated from using the term "agent" to "bot". Though in the Forta SDK you'll still encounter instances of "agent", in documentation like the README, please refer to them as bots.